packages/uiweb/src/lib/components/space/SpaceBanner/SpaceBanner.tsx

```diff
@@ -1,15 +1,320 @@
 import React from 'react';
```

Comment:
can you also add this component it in demoreact for better functionality testing?